packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -15,6 +15,8 @@ export type TextTypes = 'muted' | 'normal' | 'placeholder' | 'subdued' | 'subtle
 
 export type ColorContrastTypes = 'low' | 'high';
 
+export type IntentTypes = 'positive' | 'negative' | 'neutral' | 'notice' | 'information';
```

Comment:
We already have `Feedback`, Feedback and Intent will have the same keys right? Can we reuse that if so?

Reply:
Not sure, would be better to keep separate imo because both have different significance

Reply:
Changed to `Feedback` in https://github.com/razorpay/blade/pull/666